EIPS/eip-7488.md

```diff
@@ -0,0 +1,51 @@
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-06-30
+---
+
+## Abstract
+
+This EIP "Ground Zero" suggests to switch mainnet to permissioned to lower gas prices, transaction costs, and improve privacy. The adjustment requires protcol and governance changes. The EIP idea is braking and stopping the IPFS and Swarm store and broadcasting and instead suggests scalling down ethereum ressources, so that the community may be served in a more **sustainable way** in the future. In addition, to mitigate the currently existing problem with increasingly high gas prices, ethereum would benefit from switching to a permissioned mainnet (after successfully deploying a testnet version). This would greatly lower the number of requests and decrease the load for operating nodes. A further improvement would be to remove legacy APIs, so nodes no longer answer unnecessary public API requests. 
+
+## Motivation
+
+Since its humble beginnings, the ethereum project has been open to the public and seen many contributions. The past years, however, have made it clear that such an open ecosystem design leads to a wide array of problems such as ongoing consensus problems, increasingly high transaction costs, cases of fraud, false ownership claims of assets such as the DAO hack and resulting fork (see [EIP-779](./eip-779.md)), illegal contracts and privacy intrusion, data theft, artifical and malicious content, and many more such things. 
+
+Halting the IPFS and Swarm storage process, which enabled and multiplied above problems, as well as historically drained the ethereum ecosystem and infrastructure of important ressources and instead developing use cases for the original community would be a remedy for almost all of these problems.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+Mainnet (ID: 1) MUST be switched to permissioned. The permissioned ethereum blockchain then either applies proof-of-authority consensus (see [EIP-225](./eip-225.md)), or alternatively, a new consensus mechanism. In part, this may be achieved by halting IPFS, Swarm, and Whisper, after a successfull testnet version has been deployed, for which Goerli or Sepolia may provide blue prints to some degree.
+
+Before halting IPFS and Swarm, packages SHOULD be transfered via p2p to the original owner, including the original data.
+
+Since for some existing dApps, token holdings would likely be exposed to such a protocol change, it is REQUIRED to decide what happens to affected [ERC-721](./eip-721.md) tokens and [ERC-20](./eip-20.md) coins. A potential solution would be to gather all tokens and decide step-by-step which ones would still be valid, could be transfered and/or sold to benefit the community, and which ones should be burned.
```

Comment:
What about ERC-1155 tokens? What about other tokens?

How do you enumerate all the assets? Specifically, how to you get a list of every single ERC-20 or ERC-721 contract?

What happens when you do? Who decides? Do the validators all vote on it? If so, is there a time limit? What percentage have to vote? What format do the votes take?

Why do the assets need to be transferred at all? Will they not still work?